pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -515,7 +515,10 @@ func (o *OvirtMapper) mapArchitecture() *kubevirtv1.Machine {
 
 	// Override machine type in case custom emulated machine is specified
 	if custom, ok := o.vm.CustomEmulatedMachine(); ok {
-		machine.Type = custom
+		// do not override when i440fx which is unsupported
+		if !strings.Contains(custom, "i440fx") {
```

Comment:
Shouldn't it be part of VM validation (blocking or warning)? Will the target VM work properly if we ignore `CustomEmulatedMachine`?

Reply:
Interesting question. Machine type is not mandatory so we should not block this in validation. Setting the value base on cpu architecture works ok.

Reply:
Make sense, let's add validation rule and log that we ignore.